tests/integration/cheatcodes/expect_call/expect_call_test.cairo

```diff
@@ -0,0 +1,87 @@
+%lang starknet
```

Comment:
[Testing] A test case checking failure when there was no call despite expecting it is needed.

Reply:
[expect but not found test case](https://github.com/software-mansion/protostar/pull/1060/commits/fe139f546ac275f69f4dee9f972045a3bd40ee65)